Add support for building on GNU Hurd

This build has failed in Debian: https://buildd.debian.org/status/fetch.php?pkg=libzt&arch=hurd-i386&ver=0.2-1&stamp=1585351409&raw=0

```
 dpkg-source --before-build .
dpkg-buildpackage: info: host architecture hurd-i386
 /usr/bin/fakeroot debian/rules clean
dh clean
   dh_auto_clean
	make -j1 distclean
make[1]: Entering directory '/<<PKGBUILDDIR>>'
GNUmakefile:30: .makefiles/Makefile.GNU.mk: No such file or directory
make[1]: *** No rule to make target '.makefiles/Makefile.GNU.mk'.  Stop.
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
dh_auto_clean: error: make -j1 distclean returned exit code 2
debian/rules:4: recipe for target 'clean' failed
make: *** [clean] Error 255
/bin/fakeauth: Error 2 for child 18140
/bin/settrans: Error 2 for child 18139
```

--- a/.makefiles/Makefile.GNU.mk
+++ b/.makefiles/Makefile.GNU.mk
@@ -1,0 +1,18 @@
+# Copyright 2019-2020 Zygmunt Krynicki.
+#
+# This file is part of libzt.
+#
+# Libzt is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License.
+#
+# Libzt is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Libzt.  If not, see <https://www.gnu.org/licenses/>.
+
+# Plot twist!
+include $(srcdir)/.makefiles/Makefile.Linux.mk

